src/hooks/useCatalogResults/useCatalogResults.test.ts

```diff
@@ -47,98 +38,32 @@ describe("useCatalogResults", () => {
   afterEach(cleanup);
 
   const renderUseCatalog = (options: UseCatalogResultsOptions) =>
-    renderHook(() => useCatalogResults(options));
+    renderHook(() => useCatalogResults(options), { wrapper: SearchProvider });
 
-  it("returns pageLimit, results, displayable, loadind, and error state", () => {
+  it("returns page, pageLimit, results", () => {
     const { result } = renderUseCatalog(defaultOptions);
-    const { loading, error, pageLimit, results, displayable } = result.current;
+    const { pageLimit, results, page } = result.current;
 
-    expect(typeof loading).toBe("boolean");
-    expect(error).toBeUndefined();
     expect(typeof pageLimit).toBe("number");
+    expect(typeof page).toBe("object");
     expect(typeof results).toBe("object");
-    expect(typeof displayable).toBe("object");
-  });
-
-  it("filters results by query", () => {
```

Comment:
This hook no longer writes it's own search / filter / sort implementation so these tests don't belong here